### DISCUSSION TITLE
problems adding input fields for my secret integer

### DISCUSSION THREAD

**Discussion by ogueji** [NillionNetwork/discussions#131] [General]
hi please im having diffucilties adding input fields for secret integers on my blind nillion app im building. please help...

**Comment by wwwehr** [comment]
hi @ogueji  - can you share the code or any information about how you are using the nillion client?

best
-nicholas

**Reply by ogueji** [reply]
okay so im trying to build a blind app as im trying to participate in the nillion hacktoberfest. i think ive done the most part of the task but im finding it hard to setup input fields for my secret integers into my blind app. im confused as to where it should be put.

<img width="956" alt="image" src="https://github.com/user-attachments/assets/4670020b-6077-4f1f-b699-4cb85a0b9ab1">

**Reply by wwwehr** [reply]
hi @ogueji 

can you elaborate on what your app is trying to do? what kind of data are you storing? are you building on top of the starter project?

best
